src/OpenLoco/openloco.vcxproj

```diff
@@ -333,4 +335,4 @@
     <_NuGetTargetFallbackMoniker>$(_NuGetTargetFallbackMoniker);native,Version=v0.0</_NuGetTargetFallbackMoniker>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
\ No newline at end of file
+</Project>

```

Comment:
An EOL was introduced here. Did you edit the file outside of VS?

Reply:
No. I tend to find it goes in and out depending on the weather annoyingly.

Reply:
I don't mind either way. Just thought it might lead to some git conflict if it differs again.